action.yml

```diff
@@ -14,6 +14,9 @@ inputs:
       ideas.
     default: stable-2.11
     required: true
+  docker-image:
```

Comment:
Not sure if using `docker` is a good idea. Let's be agnostic and use "container". After all, it can also work with Podman under certain circumstances.

Reply:
docker is the option coming from ansible-test itself, so it looks sensible to keep aligned. Later if podman is supported, we can follow whatever policy ansible-test team come with.

Reply:
Well, it's not capital-S-supported but it works if the `docker` binary actually points at `podman` in the env. Having a generic name could reduce the compatibility problems when it comes to it.

Reply:
But I guess I won't insist on this. After all, there are new more flexible CLI args that probably need a `v2` anyway. Plus, maybe when we finally take ownership over `ansible-test`, I'll think about a `v3`.

Reply:
if a hack is already possible to have it working with `podman`, then I agree with your suggestion `container` or `test-container` or in this idea.

Reply:
It is possible but is not integrated into this action at the moment.

Reply:
I think I'll leave it as is for now.